src/image_streamer.cpp

```diff
@@ -39,7 +39,7 @@ void ImageTransportImageStreamer::start()
   for (auto topic_and_types : tnat) {
     if (topic_and_types.second.size() > 1) {
       // explicitly avoid topics with more than one type
-      break;
+      continue;
```

Comment:
the comment should be updated, then

Reply:
Changed it slightly to "skip over topics with more than one type", which I felt was a little more descriptive of what's actually happening.